app/views/includes/taxFreeAmountSummary.scala.html

```diff
@@ -65,16 +65,16 @@ <h3 class="heading-small section--narrow flush" id="summaryTable@{itemIdx.toStri
                                     )
                                 </dd>
                                 } else @{if(oneOrMoreLinksPresent) {
-                                    <dd class="cya-change" aria-hidden="true"></dd>
+                                    <dd class="govuk-summary-list__value cya-change" aria-hidden="true"></dd>
                                 }}
                             }
                         </div>
                     }
                 </dl>
                 } else {@if(item.rows.length == 1) {
 
-                <dl id="summaryTable@{itemIdx.toString}Body" class="govuk-check-your-answers cya-questions-long govuk-check-your-answers--tax-summary @if(itemIdx == items.length && item.rows.length == 1){subsection highlight heading-small} else {flush}">
-                    <div id="summaryTable@{itemIdx.toString}Row1">
+                <dl id="summaryTable@{itemIdx.toString}Body" class="govuk-summary-list govuk-check-your-answers cya-questions-long govuk-check-your-answers--tax-summary @if(itemIdx == items.length && item.rows.length == 1){subsection xxxhighlight heading-small} else {flush}">
```

Comment:
is xxxhighlight correct? Never seen it before

Reply:
yes needs removing

Reply:
highlight  introdcued a grey shade in the amount row - which does not exist in prototype so its been removed - also there a corresponding test has been removed for the highlight.